npm install osrm throws 403 error

I receive the following error on `npm install osrm`:

````bash
node-pre-gyp ERR! install error 
node-pre-gyp ERR! stack Error: 403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/osrm/v5.11.0/Release/node-v48-darwin-x64.tar.gz
node-pre-gyp ERR! stack     at Request.<anonymous> (/Users/AntonakakisJ/node_modules/osrm/node_modules/node-pre-gyp/lib/install.js:118:27)
node-pre-gyp ERR! stack     at emitOne (events.js:101:20)
node-pre-gyp ERR! stack     at Request.emit (events.js:188:7)
node-pre-gyp ERR! stack     at Request.onRequestResponse (/Users/AntonakakisJ/node_modules/osrm/node_modules/request/request.js:1074:10)
node-pre-gyp ERR! stack     at emitOne (events.js:96:13)
node-pre-gyp ERR! stack     at ClientRequest.emit (events.js:188:7)
node-pre-gyp ERR! stack     at HTTPParser.parserOnIncomingClient [as onIncoming] (_http_client.js:469:21)
node-pre-gyp ERR! stack     at HTTPParser.parserOnHeadersComplete (_http_common.js:105:23)
node-pre-gyp ERR! stack     at TLSSocket.socketOnData (_http_client.js:359:20)
node-pre-gyp ERR! stack     at emitOne (events.js:96:13)
node-pre-gyp ERR! System Darwin 16.7.0
node-pre-gyp ERR! command "/Users/AntonakakisJ/.nvm/versions/node/v6.0.0/bin/node" "/Users/AntonakakisJ/node_modules/osrm/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build=false"
node-pre-gyp ERR! cwd /Users/AntonakakisJ/node_modules/osrm
node-pre-gyp ERR! node -v v6.0.0
node-pre-gyp ERR! node-pre-gyp -v v0.6.36
node-pre-gyp ERR! not ok 
403 status code downloading tarball https://mapbox-node-binary.s3.amazonaws.com/osrm/v5.11.0/Release/node-v48-darwin-x64.tar.gz
sh: ./scripts/node_install.sh: No such file or directory
npm WARN enoent ENOENT: no such file or directory, open '/Users/AntonakakisJ/package.json'
npm WARN AntonakakisJ No description
npm WARN AntonakakisJ No repository field.
npm WARN AntonakakisJ No README data
npm WARN AntonakakisJ No license field.

npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! osrm@5.11.0 install: `node-pre-gyp install --fallback-to-build=false || ./scripts/node_install.sh`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the osrm@5.11.0 install script.
npm ERR! This is probably not a problem with npm. There is likely a
````

--- a/.npmignore
+++ b/.npmignore
@@ -13,3 +13,4 @@
 !profiles/*
 !profiles/lib/*
 !profiles/examples/*
+!scripts/node_install.sh

